### PR TITLE
update TiDB 6MB limit

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### What is changed, added or deleted? (Required)

TiDB introduced txn-entry-size-limit in v5.0 and added it to the doc #5158, but there are some outdated descriptions left.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6226
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: error-codes.md [hunk 1]
```diff
@@ -88,9 +88,9 @@ TiDB is compatible with the error codes in MySQL, and in most cases returns the
 
 * Error Number: 8025
 
-    The single Key-Value pair being written is too large. The largest single Key-Value pair supported in TiDB is 6 MB.
+    The single Key-Value pair being written is too large. The largest single Key-Value pair supported in TiDB by default is 6 MB.
```
Comment (TomShawn) [inline error-codes.md:91]:
```suggestion
    The single Key-Value pair being written is too large. The largest single Key-Value pair supported in TiDB is 6 MB by default.
```

File: error-codes.md [hunk 2]
```diff
@@ -88,9 +88,9 @@ TiDB is compatible with the error codes in MySQL, and in most cases returns the
...
 
-    If a pair exceeds this limit, you need to manually deal with this row of data to meet the 6 MB limit.
+    If a pair exceeds this limit, you need to properly adjust the configuration items in [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) to ease the restrictions.
```
Comment (TomShawn) [inline error-codes.md:93]: `txn-entry-size-limit` is an configuration item.
```suggestion
    If a pair exceeds this limit, you need to properly adjust the [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) configuration item to relax the limit.
```

File: faq/migration-tidb-faq.md [hunk 3]
```diff
@@ -138,7 +138,7 @@ The total read capacity has no limit. You can increase the read capacity by addi
 
 ### The error message `transaction too large` is displayed
 
-Due to the limitation of the underlying storage engine, each key-value entry (one row) in TiDB should be no more than 6MB.
+Due to the limitation of the underlying storage engine, each key-value entry (one row) in TiDB should be no more than 6MB. You can adjust the configuration items in the configuration file [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) and the maximum value is 120MB.
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:141]: ditto

File: storage-engine/titan-overview.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ Titan is suitable for the scenarios where a huge volume of data is written to th
 
 The prerequisites for enabling Titan are as follows:
 
-- The average size of values is large, or the size of all large values accounts for much of the total value size. Currently, the size of a value greater than 1 KB is considered as a large value. In some situations, this number (1 KB) can be 512 B. Note that a single value written to TiKV cannot exceed 6 MB due to the limitation of the TiKV Raft layer.
+- The average size of values is large, or the size of all large values accounts for much of the total value size. Currently, the size of a value greater than 1 KB is considered as a large value. In some situations, this number (1 KB) can be 512 B. Note that a single value written to TiKV cannot exceed 8 MB due to the limitation of the TiKV Raft layer. You can  adjust the configuration items in [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size) to ease the restrictions.
```
Comment (TomShawn) [inline storage-engine/titan-overview.md:30]: ditto

File: tidb-limitations.md [hunk 5]
```diff
@@ -51,7 +51,7 @@ This document describes the common usage limitations of TiDB, including the maxi
 
 | Type       | Upper limit   |
 |:----------|:----------|
-| Size       | 6 MB       |
+| Size       | 6 MB by default, you can adjust the configuration items in [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) |
```
Comment (TomShawn) [inline tidb-limitations.md:54]: ditto

**Review by TomShawn — COMMENTED** [review]
The `requires-version-specific-changes` label is missing. Please add this label in this PR and read the docs-cn PR description again to see which version needs specific changes. Thanks!

File: error-codes.md [hunk 6]
```diff
@@ -88,9 +88,9 @@ TiDB is compatible with the error codes in MySQL, and in most cases returns the
 
 * Error Number: 8025
 
-    The single Key-Value pair being written is too large. The largest single Key-Value pair supported in TiDB is 6 MB.
+    The single Key-Value pair being written is too large. The largest single Key-Value pair supported in TiDB is 6 MB by default.
 
-    If a pair exceeds this limit, you need to manually deal with this row of data to meet the 6 MB limit.
+    If a pair exceeds this limit, you need to properly adjust the [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) configuration item to relax the limit.
```
Comment (TomShawn) [inline error-codes.md:93]:
```suggestion
    If a pair exceeds this limit, you need to properly adjust the [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) configuration value to relax the limit.
```

File: faq/migration-tidb-faq.md [hunk 7]
```diff
@@ -138,7 +138,7 @@ The total read capacity has no limit. You can increase the read capacity by addi
 
 ### The error message `transaction too large` is displayed
 
-Due to the limitation of the underlying storage engine, each key-value entry (one row) in TiDB should be no more than 6MB.
+Due to the limitation of the underlying storage engine, each key-value entry (one row) in TiDB should be no more than 6MB. You can adjust the [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) configuration item and the maximum value is 120MB.
```
Comment (TomShawn) [inline faq/migration-tidb-faq.md:141]:
```suggestion
Due to the limitation of the underlying storage engine, each key-value entry (one row) in TiDB should be no more than 6MB. You can adjust the [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) configuration value up to 120MB.
```

File: storage-engine/titan-overview.md [hunk 8]
```diff
@@ -27,7 +27,7 @@ Titan is suitable for the scenarios where a huge volume of data is written to th
 
 The prerequisites for enabling Titan are as follows:
 
-- The average size of values is large, or the size of all large values accounts for much of the total value size. Currently, the size of a value greater than 1 KB is considered as a large value. In some situations, this number (1 KB) can be 512 B. Note that a single value written to TiKV cannot exceed 6 MB due to the limitation of the TiKV Raft layer.
+- The average size of values is large, or the size of all large values accounts for much of the total value size. Currently, the size of a value greater than 1 KB is considered as a large value. In some situations, this number (1 KB) can be 512 B. Note that a single value written to TiKV cannot exceed 8 MB due to the limitation of the TiKV Raft layer. You can  adjust the [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size) configuration item to relax the limit.
```
Comment (TomShawn) [inline storage-engine/titan-overview.md:30]:
```suggestion
- The average size of values is large, or the size of all large values accounts for much of the total value size. Currently, the size of a value greater than 1 KB is considered as a large value. In some situations, this number (1 KB) can be 512 B. Note that a single value written to TiKV cannot exceed 8 MB due to the limitation of the TiKV Raft layer. You can  adjust the [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size) configuration value to relax the limit.
```

File: tidb-limitations.md [hunk 9]
```diff
@@ -51,7 +51,7 @@ This document describes the common usage limitations of TiDB, including the maxi
 
 | Type       | Upper limit   |
 |:----------|:----------|
-| Size       | 6 MB       |
+| Size       | 6 MB by default, you can adjust the [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) configuration item |
```
Comment (TomShawn) [inline tidb-limitations.md:54]:
```suggestion
| Size       | 6 MB by default. You can adjust the size limit via the [`txn-entry-size-limit`](/tidb-configuration-file.md#txn-entry-size-limit-new-in-v50) configuration item. |
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

File: storage-engine/titan-overview.md [hunk 10]
```diff
@@ -27,7 +27,7 @@ Titan is suitable for the scenarios where a huge volume of data is written to th
 
 The prerequisites for enabling Titan are as follows:
 
-- The average size of values is large, or the size of all large values accounts for much of the total value size. Currently, the size of a value greater than 1 KB is considered as a large value. In some situations, this number (1 KB) can be 512 B. Note that a single value written to TiKV cannot exceed 6 MB due to the limitation of the TiKV Raft layer.
+- The average size of values is large, or the size of all large values accounts for much of the total value size. Currently, the size of a value greater than 1 KB is considered as a large value. In some situations, this number (1 KB) can be 512 B. Note that a single value written to TiKV cannot exceed 8 MB due to the limitation of the TiKV Raft layer. You can  adjust the [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size) configuration value to relax the limit.
```
Comment (youjiali1995) [inline storage-engine/titan-overview.md:30]:
```suggestion
- The average size of values is large, or the size of all large values accounts for much of the total value size. Currently, the size of a value greater than 1 KB is considered as a large value. In some situations, this number (1 KB) can be 512 B. Note that a single value written to TiKV cannot exceed 8 MB due to the limitation of the TiKV Raft layer. You can adjust the [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size) configuration value to relax the limit.
```

**Pull request merged**